src/main/java/net/mcreator/ui/workspace/WorkspacePanel.java

```diff
@@ -724,59 +724,7 @@ public WorkspacePanel(final MCreator mcreator) {
 
 		but3.addMouseListener(new MouseAdapter() {
 			@Override public void mouseClicked(MouseEvent e) {
-				if (but3.isEnabled()) {

```

Comment:
Why did you remove default left click action?

Reply:
for this delete button, as it was used 2 places, i prefered use it as a Method or else i would have to write this big section of codes twice and would have took too much space